core/templates/components/button-directives/primary-link-button.component.ts

```diff
@@ -13,59 +13,54 @@
 // limitations under the License.
 
 /**
- * @fileoverview Component for the primary buttons displayed on static pages.
+ * @fileoverview Component for primary link buttons displayed on static pages.
  */
 
-import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Component, Input, Output, EventEmitter, OnInit } from '@angular/core';
 import { downgradeComponent } from '@angular/upgrade/static';
-import { WindowRef } from
-  'services/contextual/window-ref.service';
-import './primary-button.component.css';
+import './primary-link-button.component.css';
 
 
 @Component({
-  selector: 'oppia-primary-button',
-  templateUrl: './primary-button.component.html',
-  styleUrls: ['./primary-button.component.css']
+  selector: 'oppia-primary-link-button',
+  templateUrl: './primary-link-button.component.html',
+  styleUrls: ['./primary-link-button.component.css']
 })
-export class PrimaryButtonComponent {
+export class PrimaryLinkButtonComponent implements OnInit {
   @Input() buttonText: string = '';
```

Comment:
Can you put comments above each of these to explain what they mean? I think this is particularly useful for common components, so feel free to go into as much detail as needed (we should probably be doing this for all other interfaces too).

Reply:
By each of these, are you referring to the parameters like buttonText and customClasses?

Reply:
Yea, that's right.

Reply:
Done